src/Defra.Trade.ReMoS.AssuranceService.API.Domain/Models/SelfServeAddEstablishmentMessage.cs

```diff
@@ -19,4 +13,17 @@ public record TradePartyWithLogicsLocationData
     public Guid Id { get; init; }
     public Guid OrgId { get; init; }
     public LogisticsLocationData? LogisticsLocation { get; init; }
+}
+
+[ExcludeFromCodeCoverage]
```

Comment:
Are we able to get these covered by any unit tests?

Reply:
Added.